docs/src/using-turing/advanced.md

```diff
@@ -60,8 +59,16 @@ Distributions.minimum(d::CustomUniform) = 0.
 Distributions.maximum(d::CustomUniform) = 1.
 ```
 
-and `Bijectors.jl` will ensure that everything just works.
+and `Bijectors.jl` will return a default `Bijector` called `TruncatedBijector` which makes use of `minimum` and `maximum` derive the correct transformation.
+
+Internally, Turing basically does the following when it needs to convert a constrained distribution to an unconstrained distribution, e.g. when sampling using `HMC`:
+```julia
+b = bijector(dist)
+transformed_dist = transformed(dist, b) # results in distribution with transformed support + correction for logpdf
+```
+and then we can call `rand(transformed_dist)` and `logpdf(transformed_dist, y)` where `y` is any real number, as usual.
```

Comment:
Consider adding a sentence to clarify whether

1. `rand` returns a sample in the unconstrained real space, and
2.  `logpdf(transformed_dist, y)` returns the log density for the original distribution, but with y living in the unconstrained space. 

At the moment, these points are a bit implicit, and it might help to be a bit more explicit.

Reply:
Done:)